src/doc/src/reference/unstable.md

```diff
@@ -1233,63 +1233,147 @@ cargo check -Z unstable-options -Z check-cfg-well-known-values
 * RFC: [#2906](https://github.com/rust-lang/rfcs/blob/master/text/2906-cargo-workspace-deduplicate.md)
 * Tracking Issue: [#8415](https://github.com/rust-lang/cargo/issues/8415)
 
-The `workspace-inheritance` feature allows workspace members to inherit fields
-and dependencies from a workspace.
+### The `workspace.package` table
 
-Example 1:
+*Stabilization*: This would be in [`workspaces.md`][workspaces], under
+[The `workspace.metadata` table][workspace-metadata-table]
 
-```toml
-# in workspace's Cargo.toml
-[workspace.dependencies]
-log = "0.3.1"
-log2 = { version = "2.0.0", package = "log" }
-serde = { git = 'https://github.com/serde-rs/serde' }
-wasm-bindgen-cli = { path = "crates/cli" }
-```
+The `workspace.package` table is where you define keys that can be
+inherited by members of a workspace. These keys can be inherited by
+defining them in the member package with `{key}.workspace = true`.
 
-```toml
-# in a workspace member's Cargo.toml
-[dependencies]
-log.workspace = true
-log2.workspace = true
-```
+Keys that are supported:
 
-Example 2:
+| A - I           | J - Z          |
+|-----------------|----------------|
+| `authors`       | `keywords`     |
+| `categories`    | `license`      |
+| `description`   | `license-file` |
+| `documentation` | `publish`      |
+| `edition`       | `readme`       |
+| `exclude`       | `repository`   |
+| `homepage`      | `rust-version` |
+| `include`       | `version`      |
```

Comment:
Should we call out the path behavior?
- license-file and readme are relative to the workspace root
- `include` and `exclude` are relative to your package root

Reply:
I will add it, it seems like a good thing to have